ember-app/app/controllers/application.js

```diff
@@ -24,6 +24,25 @@ var ApplicationController = Ember.Controller.extend(
 
   //Fix the need to delay event subscriptions
   subscribeDisabled: true,
+
+  //Browser Session Checker
+  boardSyncing: Ember.inject.service(),
+  checkBrowserSession: function(){
+    var lastFocus = this.get('browser-session.lastFocus');
+    var _self = this;
+    if(lastFocus >= _self.browserCheckInterval){
+      var since = new Date(new Date().getTime() - _self.browserCheckInterval);
+      this.validateCredentials().success((response)=>{
+        _self.get('boardSyncing').syncIssues(_self.get('model.board'), {since: since.toISOString()});
+      }).fail((error)=>{
```

Comment:
> I see this feature as a way to "latch" on focus and fail before any unauthorized tasks are performed to begin with.

I'm not sure what that actually gains us, though. Server-side, we're checking `logged_in?` in either case, we just have to make two requests if we have an explicit login check.

I'm curious what scenarios we have (or might have) in the app that would require different experiences when a 403 or 500 are encountered?